barretenberg/cpp/src/barretenberg/vm/avm/trace/gadgets/range_check.hpp

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include "barretenberg/vm/avm/generated/relations/range_check.hpp"
+#include "barretenberg/vm/avm/trace/common.hpp"
+#include <cstdint>
+
+enum class EventEmitter {
```

Comment:
Not using this right now, but when i start integrating, we will need this info to set selectors for inter-trace comms